src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -12,28 +13,27 @@ public boolean hasSomebodyWon() {
         return (hasPlayerWonSet(playerOne, playerTwo) || hasPlayerWonSet(playerTwo, playerOne));
     }
 
-    public boolean hasSomeoneOfPlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
-        return (playerWhoIsWinning.getSetScore() - playerWhoIsLoosing.getSetScore() == 2);
+    public boolean hasOneOfThePlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLosing) {
```

Comment:
nitpicking: maybe more grammatically correct naming -> `isTwoPointDifference(Player winningPlayer, Player losingPlayer)` ;)

Reply:
1. Since it doesn't affect the state of the instances of this class, you could have this method as a static method.
2. Since it's  used as a helper method only within this class, you could have it `private` instead of public.

Reply:
In this case though, it makes more sense to not create it as[`static`](https://docs.oracle.com/javase/tutorial/java/javaOO/classvars.html) method but as an instance non-static method:

````java
private boolean isTwoPointDifference() {
        return Math.abs(playerOne.getSetScore() - playerTwo.getSetScore()) == 2;
}
````

The `math.abs` gets the absolute value of the difference since it could be -ve.